1.Exploratory_Data_Analysis/anova_run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --mem=300G
+#SBATCH --partition=amem
+#SBATCH --qos=mem
+#SBATCH --time=24:00:00
+#SBATCH --output=sample-%
+
+module load anaconda
+
+conda activate Interstellar
+
+treatment1s=( DMSO_0.100_DMSO_0.025 )
```

Comment:
Interesting, just to confirm, you don't need to include commas between treatments right? The space between them is what separates them?

Reply:
@jenna-tomkinson yes, the command line is a different animal. Weird right?